### PR TITLE
updates sample data and refactor time conflict

### PR DESCRIPTION
**Pull request by YaleChen299**

### CODE REVIEW

**Review by YaleChen299 — COMMENTED** [review]
I have taken a thorough look. I think it it LGTM.

File: src/main/java/seedu/placebook/model/schedule/exceptions/ClashingAppointmentsException.java
Comment (nwhloo99) [inline src/main/java/seedu/placebook/model/schedule/exceptions/ClashingAppointmentsException.java:32]: Would it be better also to maybe give a '\n' so that the list wont get too long horizontally, might be easier to read vertically

File: src/test/java/seedu/placebook/testutil/AppointmentBuilder.java
Comment (nwhloo99) [inline src/test/java/seedu/placebook/testutil/AppointmentBuilder.java:36]: Good idea, just wondering if maybe its better to have an Enum like DEFAULT_TIME, so that its clearer, and maybe then we can add more values so that we can play around with more default values for testing.

**Review by nwhloo99 — COMMENTED** [review]
Should be ready to merge once we check on the stuff I mentioned.

File: src/main/java/seedu/placebook/commons/core/Messages.java
Comment (nwhloo99) [inline src/main/java/seedu/placebook/commons/core/Messages.java:18]: Oh my bad, now that we are doing the \n for the list, should add a \n here

File: src/main/java/seedu/placebook/commons/util/StringUtil.java
Comment (nwhloo99) [inline src/main/java/seedu/placebook/commons/util/StringUtil.java:85]: Just noticed this maybe should be -1, because java doesnt take the index at stringBuilder.length() - 2, if im not wrong it might cut out the last letter.

**Review by nwhloo99 — APPROVED** [review]
Once the formatting issues changed can merge.

File: src/main/java/seedu/placebook/commons/util/StringUtil.java
Comment (YaleChen299) [reply src/main/java/seedu/placebook/commons/util/StringUtil.java:85]: I just realised this needs to be `- delimiter.length()`

**Pull request merged**
